src/main/resources/application.properties

```diff
@@ -0,0 +1,14 @@
+spring.profiles.active=local
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.hibernate.ddl-auto=update
```

Comment:
In this case, if you add a new variable to the entity but forget to add the slq migration, it can lead to an error because the connection between Hibernate and Flyway will be inconsistent.